src/lib/payment-processing/checkoutSession.test.ts

```diff
@@ -0,0 +1,166 @@
+import { PUBLIC_ASSETS_PATH } from '$env/static/public';
+import { describe, expect, test } from 'vitest';
+import { CreateCheckoutSessionError, formatCheckoutSessionParams } from './checkoutSession';
+import { radomAdapter } from './providers/radom';
+import { stripeAdapter } from './providers/stripe';
+
+const requestBody = {
+  items: [
+    { id: 'clwsmtxpp0000zo6dwkmvu76n', quantity: '1' },
+    { id: 'clwsmtxrj0004zo6dsqvo4b7t', quantity: '1' }
+  ],
+  shippingAddress: {
```

Comment:
Do you mind making this fake data? I know this is just a test, but it's likely better not to be inadvertently holding employee data in the git history

Reply:
Oof. Yes. Thank you so much for catching this.